Hall-of-Fame.md

```diff
@@ -3,3 +3,25 @@
 Our "Hall of Fame" is a symbol of recognition and appreciation for all the efforts and contributions performed by the members to enhance the Windows Community Toolkit and accomplish various tasks. The list includes the name of former members who performed indelibly and left their footprints through honest dedication and determination towards the community. Their added value and positive approach to expand the community will always be valued and appreciated. 
 
 These were the community members with impeccable performance and consistency which will always be cherished. These members are always welcome to return and continue participating in the Windows Community Toolkit.
+
+## Current Hall of Fame Members
+ 
+[![Pedro Lamas](https://github.com/pedrolamas.png?size=200)](https://github.com/pedrolamas)
+[![Nikola Metulev](https://github.com/nmetulev.png?size=200)](https://github.com/nmetulev)
+[![David Catuhe](https://github.com/deltakosh.png?size=200)](https://github.com/deltakosh)
+[![Hermit Dave](https://github.com/hermitdave.png?size=200)](https://github.com/hermitdave)
+[![Shen Chauhan](https://github.com/shenchauhan.png?size=200)](https://github.com/shenchauhan)
+[![Barbara Martinez](https://github.com/Bmartin2013.png?size=200)](https://github.com/Bmartin2013)
+[![William Bradley](https://github.com/WilliamABradley.png?size=200)](https://github.com/WilliamABradley) 
+[![AVK](https://github.com/avknaidu.png?size=200)](https://github.com/avknaidu)
+[![Ibraheem Osama](https://github.com/IbraheemOsama.png?size=200)](https://github.com/IbraheemOsama)
+[![VisualByteStudios](https://github.com/VisualByteStudios.png?size=200)](https://github.com/VisualByteStudios)
+[![Richard Murillo](https://github.com/rjmurillo.png?size=200)](https://github.com/rjmurillo)
+[![Diederik Krols](https://github.com/XamlBrewer.png?size=200)](https://github.com/XamlBrewer)
+[![Chris Barker](https://github.com/cbarkerms.png?size=200)](https://github.com/cbarkerms)
+[![Harini Kannan](https://github.com/harinikmsft.png?size=200)](https://github.com/harinikmsft)
+[![Vijay Nirmal](https://github.com/Vijay-Nirmal.png?size=200)](https://github.com/Vijay-Nirmal)
+[![Ryland](https://github.com/ryalanms.png?size=200)](https://github.com/ryalanms)
+[![bixybixy2](https://github.com/bixybixy2.png?size=200)](https://github.com/bixybixy2)
```

Comment:
For now, let's just remove folks with a default image. I'll try and file a bug with GitHub about this, and we can just make a note somewhere else for folks (teams chat for Hall of Fame?) that they should make sure they have an image in their profile?